scripts/test_execution.json

```diff
@@ -10,13 +10,30 @@
     [
         {
             "Name": "vm1",
-            "V4Address": "20.0.0.1",
-            "V6Address": "2001:eb8f::1"
+            "Interfaces":
+            [
+                {
+                    "Alias": "Ethernet",
+                    "V4Address": "20.0.0.1",
+                    "V6Address": "2001:eb8f::1"
+                },
+                {
+                    "Alias": "Ethernet 2",
+                    "V4Address": "20.0.0.2",
+                    "V6Address": "2001:eb8f::2"
```

Comment:
Where do these IPv6 addresses come from?  https://www.iana.org/assignments/ipv6-unicast-address-assignments/ipv6-unicast-address-assignments.xhtml doesn't show 2001:eb8f:: as being allocated.   20.0.0.0/11 is allocated to Microsoft, but I can't find anything that says 2001:eb8f:: is allocated to Microsoft.

Reply:
Moved to link local.